user_tools/src/spark_rapids_tools/tools/qualx/qualx_main.py

```diff
@@ -220,15 +224,15 @@ def _compute_summary(results: pd.DataFrame) -> pd.DataFrame:
     # compute the fraction of app duration w/ supported ops
     # without qual tool output, this is the entire SQL duration
     # with qual tool output, this is the fraction of SQL w/ supported ops
-    summary['appDuration'] = summary['appDuration'].clip(lower=summary['Duration'])
+    summary['appDuration'] = summary['appDuration'].clip(lower=summary[label])
     summary['fraction_supported'] = (
-        summary['Duration_supported'] / summary['appDuration']
+        summary[f'{label}_supported'] / summary['appDuration']
     )
 
     # compute the predicted app duration from original app duration and predicted SQL duration
     # note: this assumes a non-SQL speedup of 1.0
     summary['appDuration_pred'] = (
-        summary['appDuration'] - summary['Duration'] + summary['Duration_pred']
+        summary['appDuration'] - summary[label] + summary[f'{label}_pred']
```

Comment:
Not sure about the meaning of this for duration_sum.  Wall clock time of app - duration_sum?    

Perhaps profiler should be enhanced to output total app duration_sum?